com.unity.render-pipelines.high-definition/Editor/AssetProcessors/MaterialPostProcessor.cs

```diff
@@ -561,6 +562,32 @@ static void ExposedDecalInputsFromShaderGraph(Material material, HDShaderUtils.S
             {
                 HDShaderUtils.ResetMaterialKeywords(material);
             }
+        }     
+
+        static void FixIncorrectEmissiveColorSpace(Material material, HDShaderUtils.ShaderID id)
+        {
+            // kEmissiveColorLDR wasn't correctly converted to linear color space.
+            // so here we adjust the value of kEmissiveColorLDR to compensate. But only if not using a HDR Color
+            const string kUseEmissiveIntensity = "_UseEmissiveIntensity";
+
+            if (material.HasProperty(kUseEmissiveIntensity) && material.GetInt(kUseEmissiveIntensity) == 1)
+            {
+                const string kEmissiveColorLDR = "_EmissiveColorLDR";
+                const string kEmissiveColor = "_EmissiveColor";
+                const string kEmissiveIntensity = "_EmissiveIntensity";
+
+                if (material.HasProperty(kEmissiveColorLDR) && material.HasProperty(kEmissiveIntensity) && material.HasProperty(kEmissiveColor))
+                {
+                    // Important:  The color picker for kEmissiveColorLDR is LDR and in sRGB color space but Unity don't perform any color space conversion in the color
+                    // picker BUT only when sending the color data to the shader... So as we are doing our own calculation here in C#, we must do the conversion ourselves.
+                    Color emissiveColorLDR = material.GetColor(kEmissiveColorLDR);
+                    Color emissiveColorLDRLinear = new Color(Mathf.LinearToGammaSpace(emissiveColorLDR.r), Mathf.LinearToGammaSpace(emissiveColorLDR.g), Mathf.LinearToGammaSpace(emissiveColorLDR.b));
```

Comment:
This is named emissiveColorLDRLinear, but we are converting **to** gamma **from** linear

Reply:
good catch :)

Reply:
fixed